modules/oe_subscriptions_anonymous/src/MailTemplate/EmailTaken.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_subscriptions_anonymous\MailTemplate;
+
+use Drupal\Core\DependencyInjection\AutowireTrait;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Prepares the "email_taken" mail.
+ */
+class EmailTaken implements ContainerInjectionInterface, MailTemplateInterface {
+
+  use StringTranslationTrait;
+  use AutowireTrait;
+
+  /**
+   * Creates a new instance of this class.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   Entity type manager.
+   */
+  public function __construct(
+    protected readonly EntityTypeManagerInterface $entityTypeManager,
+  ) {}
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getParameters(): array {
+    return [
+      'entity_type',
+      'entity_id',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getVariables(array $params): array {
+    [
+      'entity_type' => $entity_type,
+      'entity_id' => $entity_id,
+    ] = $params;
+
+    $entity = $this->entityTypeManager
+      ->getStorage($entity_type)
+      ->load($entity_id);
+
+    // @todo In theory, it is possible that the entity no longer exists.
+    return [
+      'entity_label' => $entity->label(),
+      'entity_url' => $entity->toUrl()->setAbsolute()->toString(),
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepare(array $params): array {
+    $variables = $this->getVariables($params);
+
+    $message['subject'] = $this->t('Cannot subscribe to @entity_label', [
+      '@entity_label' => $variables['entity_label'],
+    ]);
+
+    $message['body'] = $this->t(
+      '<p>Thank you for showing interest in keeping up with the updates for <a href=":entity_url">@entity_label</a>!</p>
+<p>The email address you were using to subscribe is already associated with a regular account on this website.</p>
+<p>If you still want to subscribe to content updates for this item, you should log into the website, using your existing account, and then subscribe as a regular user.</p>
+<p>If you do not want to subscribe, you can ignore this message.</p>',
```

Comment:
Message text to be reviewed and refined.

Reply:
Also, we currently have duplication in the email text:
We have one place that defines the email content when using Drupal core mailer, and another place for symfony mailer.
I am not really happy about this, but it seems changing this would be out of scope for this issue.

Reply:
Btw the `<p>` tags are converted to line breaks in non-html email, so that between two paragraphs there is always a blank line.
Imo it is fine to have these `<p>` tags, even if they are later replaced.
TBD.